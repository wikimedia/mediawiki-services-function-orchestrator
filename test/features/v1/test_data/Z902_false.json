{
	"Z1K1": "Z7",
	"Z7K1": {
		"Z1K1": "Z8",
		"Z8K1": [
			{
				"Z1K1": "Z9",
				"Z9K1": "Z17"
			},
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z40"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z802K1"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": [
						"Z1"
					]
				}
			},
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z1"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z802K2"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": [
						"Z1"
					]
				}
			},
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z1"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z802K3"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": [
						"Z1"
					]
				}
			}
		],
		"Z8K2": "Z1",
		"Z8K3": [
			"Z1"
		],
		"Z8K4": [
			{
				"Z1K1": "Z9",
				"Z9K1": "Z14"
			},
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z14"
				},
				"Z14K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z802"
				},
				"Z14K4": {
					"Z1K1": "Z6",
					"Z6K1": "Z902"
				}
			}
		],
		"Z8K5": "Z802"
	},
	"Z802K1": {
		"Z1K1": "Z40",
		"Z40K1": "Z42"
	},
	"Z802K2": [
		"Z6",
		{
			"Z1K1": "Z6",
			"Z6K1": "arbitrary ZObject"
		}
	],
	"Z802K3": {
		"Z1K1": "Z40",
		"Z40K1": "Z42"
	}
}
