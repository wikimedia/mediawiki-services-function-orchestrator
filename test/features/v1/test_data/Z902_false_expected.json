{ "Z1K1": { "Z1K1": "Z9", "Z9K1": "Z40" }, "Z40K1": { "Z1K1": "Z9", "Z9K1": "Z42" } }
